src/App.vue

```diff
@@ -161,75 +178,92 @@
         this.$root.tabTheme = to
       }
     },
+    updated() {
+      if (this.$isWebApp) this.filterLinksAndImages(window.document.body)
+    },
     mounted () {
       this.$nextTick(() => this.updateAppIcon())
+      const user_path_target = window.location.pathname
 
       this.$localData.root.TABS_SWITCH_TO()
+      // Switch to the last tab (good) but replaces history (so we use the previously captured value)
 
-      electron.webFrame.setZoomFactor(1)
+      if (user_path_target != this.$localData.root.activeTabObject.url) {
+        this.$logger.warn("Navigating user to", user_path_target)
+        this.$nextTick(() => {
+          // this.$localData.root.TABS_PUSH_URL(user_path_target)
+          this.$localData.root.TABS_NEW(user_path_target)
+        })
+      } else {
+        // this.$logger.debug(this.$localData.root.activeTabObject.url, "and", user_path_target, "match")
 
-      // Ask for a fresh copy of the archive
-      // Root must exist to receive it, so this calls from inside the app
-      electron.ipcRenderer.send("RELOAD_ARCHIVE_DATA") 
```

Comment:
This turns out to have always been wrong, but because they're different threads the race condition never happened.